Code/Framework/AzFramework/AzFramework/TargetManagement/TargetManagementComponent.cpp

```diff
@@ -721,22 +721,21 @@ namespace AzFramework
 
     void connect_target([[maybe_unused]] const AZ::ConsoleCommandContainer& arguments)
     {
-        for (auto& networkInterface : AZ::Interface<AzNetworking::INetworking>::Get()->GetNetworkInterfaces())
+        AzNetworking::INetworkInterface* networkInterface =
+            AZ::Interface<AzNetworking::INetworking>::Get()->RetrieveNetworkInterface(AZ::Name("TargetManagement"));
+        if (networkInterface)
         {
-            if (networkInterface.first == AZ::Name("TargetManagement"))
-            {
-                const uint16_t serverPort = target_port;
+            const uint16_t serverPort = target_port;
 
-                AzNetworking::ConnectionId connId = networkInterface.second->Connect(
-                    AzNetworking::IpAddress(TargetServerAddress, serverPort, AzNetworking::ProtocolType::Tcp));
-                if (connId != AzNetworking::InvalidConnectionId)
-                {
-                    AzFrameworkPackets::TargetConnect initPacket;
-                    initPacket.SetPersistentId(AZ::Crc32(Platform::GetPersistentName()));
-                    initPacket.SetCapabilities(Neighborhood::NEIGHBOR_CAP_LUA_VM | Neighborhood::NEIGHBOR_CAP_LUA_DEBUGGER);
-                    initPacket.SetDisplayName(Platform::GetPersistentName());
-                    networkInterface.second->SendReliablePacket(connId, initPacket);
-                }
+            AzNetworking::ConnectionId connId =
+                networkInterface->Connect(AzNetworking::IpAddress(TargetServerAddress, serverPort, AzNetworking::ProtocolType::Tcp));
+            if (connId != AzNetworking::InvalidConnectionId)
```

Comment:
any error if it is an invalidconnection? tell users to make sure the addr and port is correct.

Reply:
No this is actually by design. TargetManagement polls to find its host so failing to find it is an ordinary occurrence. This would be Invalid every time Editor attempts to connect while the Lua IDE isn't up for example.